tests/build.gradle

```diff
@@ -20,15 +19,10 @@ repositories {
     mavenCentral()
 }
 
-tasks.named('compileTestKotlin') {
-    dependsOn ':mockito-kotlin:jar'
-}
-
 dependencies {
-    implementation files("${rootProject.projectDir}/mockito-kotlin/build/libs/mockito-kotlin-${version}.jar")
+    implementation "org.mockito.kotlin:mockito-kotlin"
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation "org.mockito:mockito-core:5.7.0"
```

Comment:
13\. also removed this dependency, because it's now coming again from #498, and #499 can only be considered superseded with this line removed.